### ISSUE TITLE
Geometry misses the GL built-in attributes

### ISSUE THREAD

**Issue by ptsans** [pixijs/pixijs#7526]
When I perform operations on the built-in variable **gl_VertexID** or **gl_InstanceID** in the vertex shader, I get an error:

![Screenshot_20210606_234139](https://user-images.githubusercontent.com/18110232/120930716-06640b00-c721-11eb-887d-d006cbebd598.png)

Dumb example of vertex shader:

```GLSL

#version 300 es

precision mediump float;

in vec2 aVertexPosition;
in vec2 aUvs;

uniform mat3 translationMatrix;
uniform mat3 projectionMatrix;

out vec2 vUvs;

float foo(float a) {
    return a + 1.0;
}

void main() {

    float bar = foo(float(gl_VertexID));

    vUvs = aUvs;
    gl_Position = vec4((projectionMatrix * translationMatrix * vec3(aVertexPosition, 1.0)).xy, 0.0, 1.0);
}

```
![Screenshot_20210606_235811](https://user-images.githubusercontent.com/18110232/120931181-1aa90780-c723-11eb-8bcd-5ffea87b36bb.png)

**Comment by eXponenta** [comment]
It should not have it. This is built in attributes and it should not presented as attributes. This is very strange that shader return reserved name as attribute location. Which browser you use?

**Comment by ptsans** [comment]
@eXponenta I'm using Google Chrome 91.0.4472.77 (Official Build) (64-bit). Pixi version is 6.0.4. [Example with error above.](https://www.pixiplayground.com/#/edit/XVl6doQTGKqA3hmrj6lg_)

**Comment by ptsans** [comment]
With [example](https://jsfiddle.net/cbzv62qL/1/) from webgl2fundamentals it works as expected for me.

**Comment by eXponenta** [comment]
Oh...
WebGL return built-in attributes in shader location data and pixi not skip it.
@ivanpopelyshev @bigtimebuddy 
This is regression for es300. 
We should filter `gl_` attributes locations.

**Comment by ShukantPal** [comment]
Fixed in #7527
